### DISCUSSION TITLE
offline docs

### DISCUSSION THREAD

**Discussion by phneoix** [zauberzeug/nicegui#701] [Q&A] [answered]
### Question

Hi,
Can anyone point where can i download offline docs of nicegui (pdf of html) as i stay in area of poor internet connectivity.

Regards

**Comment by falkoschindler** [comment, marked as answer]
Currently we have no PDF version of the documentation ready for download.
But you can checkout the repository and start main.py. This should render the website wherever you are.

**Reply by rodja** [reply]
Or start our docker container. By default it also provides the website with full documentation:

```bash
docker run -it --rm -p 8080:8080  zauberzeug/nicegui
```

This will open the documentation at http://localhost:8080.

**Reply by phneoix** [reply]
Thanks. 
Working fine.

**Comment by maxamly** [comment]
Hey! I think it might be a great time to revisit this idea in the current AI era.
Most LLMs are already familiar with NiceGUI 1., but they don’t yet understand version 2. and beyond.

NiceGUI has the potential to become the shadcn/ui of Python (in a good way!) - clean, expressive, and widely adopted.

A single-file documentation version would be super helpful both for LLMs and for humans who want a quick, searchable reference

Thanks for the amazing work you’re doing!

**Reply by evnchn** [reply]
check out https://nicegui.io/static/sitewide_index.json

#4688, coded by yours truly
